.travis.yml

```diff
@@ -76,19 +76,19 @@ matrix:
       osx_image: xcode9.4
       env: >
         TOOLCHAIN=osx-10-13-make-cxx14
-        PROJECT_DIR=examples/foo
+        PROJECT_DIR=examples/ncursesw
 
     - os: osx
       osx_image: xcode9.4
       env: >
         TOOLCHAIN=osx-10-13-cxx14
-        PROJECT_DIR=examples/foo
+        PROJECT_DIR=examples/ncursesw
 
     - os: osx
       osx_image: xcode9.4
       env: >
         TOOLCHAIN=ios-nocodesign-11-4-dep-9-3
-        PROJECT_DIR=examples/foo
+        PROJECT_DIR=examples/ncursesw
```

Comment:
>  Here is the links to the Travis/AppVeyor with status "All passed":

iOS toolchain is failing:
- https://travis-ci.org/tstack/hunter/jobs/553429491

Please exclude it. Exclude other failing toolchains too.